server/app/modules/join_hunt/router.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter
+from app.modules.join_hunt.service import service
+from app.modules.join_hunt.router_models import Join_Hunts_Model
+
+router = APIRouter()
+
+@router.post(
+  "/join_hunt",
+  status_code=200,
+  response_model=Join_Hunts_Model
+)
+async def load_upcoming_hunts():
+  result = await service.get_upcoming()
+  return Join_Hunts_Model(
+    message="hunts_to_join",
```

Comment:
This message field should probably be an Enum and not a string. Check out https://github.com/PraxisAppDev/AFTERHOURS/blob/b00f820507e8bddecb4498fdc2a07270867a3e55/server/app/modules/users/auth/router_models.py#L131

When you make it an enum, reference the appropriate Enum constant instead of using string literals.